features/private-click-measurement.yml

```diff
@@ -0,0 +1,6 @@
+name: Private click measurement
+description: Private Click Measurement, or PCM, is an API for measuring ad clicks across websites.
+spec: https://privacycg.github.io/private-click-measurement/
+compat_features:
+  - api.HTMLAnchorElement.attributionSourceId
+  - html.elements.a.attributionsourceid
```

Comment:
Weird how BCD doesn't have data for the `attributiondestination` bit. I've filed https://github.com/mdn/browser-compat-data/issues/25595